src/piadm.sh

```diff
@@ -991,10 +1110,45 @@ refresh_or_disable_pool() {
 		err "Pool $pool is not bootable, and cannot be disabled or refreshed"
 
 	if [[ "$flag" == "-d" ]]; then
+		if [[ "$TRITON_HN" == "yes" ]]; then
+			# NOTE: Disabling a Head Node's boot pool may
+			# This might be where we call sdc-usbkey with
+			# a forcing-flag to seek a stick!  Or at least to
+			# scan for alternate bootable pools.
```

Comment:
Right now it errors out (read below the comment)... you cannot disable the pool that you booted from.  Are you suggesting we *allow* it, or at least allow-with-safety-belts?

Reply:
I just mean that the comment itself doesn't read properly. In particular, line 1114 seems like it can be entirely removed? Or there's a whole line (or more?) missing. Lines 1115-1117 are ok, but 1114 is missing something.

Reply:
Oh I get it.  Yeah, it's a sentence fragment.  I either cut a line or didn't finish this.  Will rework.